system-int-test/test/cmr/system_int_test/search/collection_relevancy/collection_relevancy.clj

```diff
@@ -30,6 +30,7 @@
 (deftest relevancy-temporal-ranges
```

Comment:
I think this could impact other tests. We should probably put these 3 config change calls into a fixture or set the values back to the default at the end of the test.